src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserService {
```

Comment:
remove wasteful methods
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/20326858/6e4c9f55-c250-4c78-be50-e306627e5d26)

Reply:
not fixed